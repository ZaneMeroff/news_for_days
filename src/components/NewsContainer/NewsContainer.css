
.news-container {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  margin-left: 15%;
  margin-top: 60px;
}

.error-message {
  font-size: 110%;
  height: 100vh;
  margin: 50px 0px 0px 50px;
  width: 30vw;
}
