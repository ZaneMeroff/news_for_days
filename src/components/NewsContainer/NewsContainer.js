import React from 'react';
import NewsArticle from '../NewsArticle/NewsArticle.js';
import App from '../App/App.js';
import './NewsContainer.css';

const NewsContainer = prop => {
  return (
    <div className='news-container'>
      {
        prop.localNews.map(article => (
        <NewsArticle
          image={article.img}
          headline={article.headline}
          description={article.description}
        />
        ))
      }
    </div>
  )
}

export default NewsContainer;
