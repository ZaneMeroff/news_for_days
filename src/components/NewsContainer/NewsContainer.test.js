import React from 'react';
import NewsContainer from '../NewsContainer/NewsContainer';
import { shallow } from 'enzyme';

describe('NewsContainer', () => {

  it('should be able to render news articles to page', () => {
    const wrapper = shallow(<NewsContainer
      news={[]}
      filterSearchResults={jest.fn()}
      />)
    expect(wrapper).toMatchSnapshot();
  });

});
