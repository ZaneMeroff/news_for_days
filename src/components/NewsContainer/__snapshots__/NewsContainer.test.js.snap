// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsContainer should be able to render news articles to page 1`] = `
<article
  className="news-container"
>
  <SearchForm
    filterSearchResults={[MockFunction]}
  />
  <p
    className="error-message"
  >
    sorry, your search did not return any results
  </p>
</article>
`;
